test/fixtures/basic-import/expected.js

```diff
@@ -1,5 +1,7 @@
+function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (obj != null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) newObj[key] = obj[key]; } } newObj.default = obj; return newObj; } }
```

Comment:
babelHelpers are included either way. User's have the option of using `external-babel-helpers` in which case babel will take care of not inlining this, but pulling it from wherever the user provides it.

https://babeljs.io/docs/plugins/external-helpers/